l10n/en/firefox/browsers/compare/opera.ftl

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/browsers/compare/opera/
+
+# Page title
+compare-opera-firefox-vs-opera = { -brand-name-firefox } vs Opera
+
+# HTML page description
+compare-opera-read-our-comparison = Read our comparison of { -brand-name-firefox } and Opera browsers on features, privacy, and ease-of-use.
+
+# Variables:
+# $fingerprint (url) - link to https://www.mozilla.org/firefox/features/block-fingerprinting/
+compare-opera-firefoxs-default-privacy-settings = { -brand-name-firefox }’s default privacy settings are stronger than Opera’s – and we have more features to prevent websites from tracking you, such as <a { $fingerprint }>fingerprint blocking</a>.
+compare-opera-firefox-offers-a-wide-range-of = { -brand-name-firefox } offers a wide range of customization options, including the ability to move menus and toolbars to different locations on the browser window. Opera’s interface is less customizable.
+compare-opera-since-we-dont-have-to-make = Since we don’t have to make shareholders happy, we can focus on making <em>you</em> happy and always put your privacy and convenience first.
+
+# Variables:
+# $howto (url) - link to https://support.mozilla.org/kb/import-bookmarks-and-other-data-microsoft-edge
+compare-opera-switching-to-firefox-is-easy = Switching to { -brand-name-firefox } is easy and fast - import your Opera bookmarks, your passwords, history, and preferences with one click and immediately be ready to use { -brand-name-firefox }. Here’s <a { $howto }>how to use the Import button to switch</a>.
```

Comment:
**Nit:** em-dash to be consistent with the string above using it as well (`compare-opera-firefoxs-default-privacy-settings`
```suggestion
compare-opera-switching-to-firefox-is-easy = Switching to { -brand-name-firefox } is easy and fast – import your Opera bookmarks, your passwords, history, and preferences with one click and immediately be ready to use { -brand-name-firefox }. Here’s <a { $howto }>how to use the Import button to switch</a>.
```